example/src/NativeBannerExample.tsx

```diff
@@ -34,9 +39,13 @@ const NativeBannerExample = ({
                 <AdView
                     adUnitId={adUnitId}
                     adFormat={AdFormat.BANNER}
-                    style={styles.banner}
+                    style={{...styles.banner, ...bannerSize}}
+                    adaptiveBannerEnabled={adaptiveBannerEnabled}
                     onAdLoaded={(adInfo: AdInfo) => {
                         log('Banner ad loaded from ' + adInfo.networkName);
+                        if (adaptiveBannerEnabled) {
+                             setBannerSize({width: adInfo.size?.width, height: adInfo.size?.height});
```

Comment:
Is `adInfo.size?.width` the width of the screen which is what the demo app already uses?

Reply:
This is a bug, even with the current version.   Fixed to use the screen width.